src/scrabble.js

```diff
@@ -1,6 +1,63 @@
 class Scrabble {
   // Write your implementation here
 
+  constructor (wordValue) {
+    this.wordValue = wordValue
+    this.isDouble = false
+    this.isTriple = false
+    this.obj = {
```

Comment:
It's interesting. You used object to wrap the letters. I used the For Loop. Yours look easier and I'm going to try that next time I come across a similar exercise.